source/client-side-encryption/etc/validate-corpus.py

```diff
@@ -0,0 +1,35 @@
+import json
+import sys
+
+description = """Validate that the same deterministically encrypted values in the corpus encrypt to the same ciphertext in the encrypted corpus.
```

Comment:
Instead of having a more complicated O(n^2) check in drivers to check all deterministically encrypted values that  should be equal, I decided to opt for a python script to check `corpus-encrypted.json` once.

That way drivers check the one corresponding field, e.g. for `aws_double_det_explicit_id` just check that the ciphertext produced is an exact match for the `aws_double_det_explicit_id` field in `corpus-encrypted.json`. This makes it simpler to add the padding cases without adding more special cases to the check in the spec.